src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)].toString();

```

Comment:
```suggestion
        return colors[random.nextInt(colors.length)].name();
```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/12864128/bba2f2f0-d019-47da-a387-71c000ebf01a)